modules/core/.js/src/main/scala/io/chrisdavenport/fuuid/PlatformSpecificMethods.scala

```diff
@@ -0,0 +1,8 @@
+package io.chrisdavenport.fuuid
+
+import cats.ApplicativeError
+
+object PlatformSpecificMethods {
+  def nameBased[F[_]]: (FUUID, String, ApplicativeError[F, Throwable]) => F[FUUID] = (_, _, AE) =>
```

Comment:
I'm using a function here, because there's no way to suppress the unused parameter warning for methods.